src/cladetime/__init__.py

```diff
@@ -2,6 +2,10 @@
 
 import structlog
 
+from cladetime.cladetime import CladeTime
```

Comment:
This line lets people do `from cladetime import CladetTme` instead of `from cladetime.cladetime import CladeTime`

Reply:
you know this is my favorite thing